src/stylesheets/application.scss

```diff
@@ -0,0 +1,9 @@
+@import 'bootstrap_manifest';
+
+@import 'variables';
+
+//@import 'mixins/*';
```

Comment:
I think we can remove these unused importing.

Reply:
What do you think?

Reply:
I just kept it to show if the structure and precedence of the import is ok or not. In the next PR this will be replace with real imports 😅

Reply:
👍